server/src/auth/handler/auth.handler.ts

```diff
@@ -12,7 +12,8 @@ export class AuthHandler {
   public login = async (req: SocoboRequest, res: Response, next: NextFunction): Promise<void> => {
     try {
       const result = await this._getExtractedRequestBodyResult(req, "login(..)");
-      const loginResult = await this._authService.login(result);
+      const pwIsToken = req.query.token;
```

Comment:
is `token` query param a boolean? what's the differenz if this flag is set?

Reply:
If the token query param is true I can login directly with a token instead of a password. See above comment

Reply:
this is not a token. its the password hash value you use. or get i wrong?

Reply:
I just call these things token since a token is simply a hashed value. I didn't mean the auth token here.